packages/Webkul/Admin/src/Resources/lang/es/app.php

```diff
@@ -1511,122 +1511,122 @@
                 ],
 
                 'create'    => [
```

Comment:
Please Remove extra space before double arrow there should be only one space. 
For example:
'create' => [
],